dlclive/dlclive.py

```diff
@@ -478,3 +486,154 @@ def close(self):
         self.is_initialized = False
         if self.display is not None:
             self.display.destroy()
+
+
+class MultiAnimalDLCLive(DLCLive):
+    def __init__(
+        self,
+        model_path,
+        n_animals,
+        n_multibodyparts,
+        track_method: str = "box",
+        min_hits: int = 1,
+        max_age: int = 1,
+        sim_threshold: float = 0.6,
+        resize: Optional[float] = None,
+        convert2rgb: bool = True,
+        processor: Optional['Processor'] = None,
+        display: typing.Union[bool, Display] = False,
+        pcutoff: float = 0.5,
+        display_radius: int = 3,
+        display_cmap: str = "bmy",
+    ):
```

Comment:
needs docstring! Since inherits from `DLCLive` but the signature differs, need to also document changes in calling convention. Also need to document any new attrs